### ISSUE TITLE
disruption of carpal arcs

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10730]
**Preferred term label:**
disruption of carpal arcs

**Synonyms**

**Definition (free text, please give PubMed ID)**
Disruption of Gilula's carpal arcs, with carpal bones not lined up in the proper orientation (see https://radiopaedia.org/articles/gilula-three-carpal-arcs)

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Perilunate dislocation (PLD)
Carpal instability nondissociative (CIND)

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6001394
- label: Disruption of carpal arcs
- parents: [Abnormal wrist morphology (HP:0025798)](https://hpo.jax.org/app/browse/term/HP:0025798)
- pmid: PMID:37511801
- def: The arcs of the Gilula are used on PA wrist radiographs as part of assessing the normal alignment of the carpus. The 1st arc (red) outlines the proximal surfaces and the 2nd arc (yellow) the distal surfaces of the scaphoid, lunate and triquetrum. The 3rd arc (white) outlines the proximal surfaces of the capitate and hamate. Disruption of carpal arcs (i.e., the arcs appear broken or discontinuous on radiographs) indicates a fracture or ligamentous injury in the wrist.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10730
- synonyms: Disruption of Gilula carpal arcs(NONE)
